webpack/makeconfig.js

```diff
@@ -75,9 +75,9 @@ module.exports = function(isDevelopment) {
       }, {
         exclude: /node_modules/,
         loaders: isDevelopment ? [
-          'react-hot', 'babel-loader'
+          'react-hot', 'babel-loader?stage=0'
```

Comment:
I think it would be better to include `.babelrc` file and enable there only those features that we need from the ES7 proposals.